code-scanning/anchore.yml

```diff
@@ -28,20 +28,21 @@ jobs:
     permissions:
       contents: read # for actions/checkout to fetch code
       security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
-      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status 
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout the code
+    - name: Check out the code
       uses: actions/checkout@v3
     - name: Build the Docker image
       run: docker build . --file Dockerfile --tag localbuild/testimage:latest
-    - name: Run the Anchore scan action itself with GitHub Advanced Security code scanning integration enabled
-      uses: anchore/scan-action@b08527d5ae7f7dc76f9621edb6e49eaf47933ccd
+    - name: Run the Anchore Grype scan action
+      uses: anchore/scan-action@36bd2c8c3b017db8c98ffbb5f866533890b7aef1
```

Comment:
This commit sha doesn't seem to belong to any releases. Is it the correct sha you want?

Reply:
I've updated the the SHA of the newly released [v3.3.0](https://github.com/anchore/scan-action/releases/tag/v3.3.0)